test/e2e-tests.sh

```diff
@@ -14,6 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
+
+# FIXME(vdemeester) Remove once 0.11 is released
+export RELEASE_YAML=https://github.com/tektoncd/pipeline/releases/download/v0.11.0-rc2/release.yaml
```

Comment:
It might be possible to use presents https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#presets but since it's a temporary situation  it's fine like this.
If we hit this in future we might look for an alternative solution - hopefully in the meanwhile this will be Tekton driven :)